nestkernel/nest_types.h

```diff
@@ -32,6 +32,19 @@
 // Generated includes:
 #include "config.h"
 
+#ifdef _OPENMP
+  #define NEST_PARALLEL_FOR_THREAD(t) \
+    _Pragma("omp parallel") \
+    { \
+      size_t t = kernel().vp_manager.get_thread_id();
```

Comment:
@jakobj You are right, this should definitely be `const`: the thread number should not change inside a thread-parallel block. I believe the reason for declaring this `size_t` was that `get_num_threads()` below returns `size_t` and one then runs into signed/unsigned comparison warnings. In general, I would advocate moving away from NEST-specific renames of PODs, so I suggest we keep `size_t` here.